src/sections/Meshery/Meshery-integrations/IntegrationsGrid.js

```diff
@@ -288,8 +288,8 @@ const IntegrationsGrid = ({ category, count }) => {
                           : integrationIcon
                       }
                       alt={item.frontmatter.title}
-                      height={70}
-                      width={70}
+                      height={55}
```

Comment:
why this change?

Reply:
Actually I had make those changes for fixing the height and width AMD logo in the integration grid section page. So, this was for a prior issue for which I was assigned for. 

This was the issue 

[Bug] Name of the Maintainers isn't visible in dark mode. #5314


I am new to open source contribution so correct me if I had not followed the guidelines well.